src/main/java/fi/aalto/cs/apluscourses/model/Authentication.java

```diff
@@ -3,8 +3,5 @@
 import fi.aalto.cs.apluscourses.utils.CoursesClient;
 
 public interface Authentication extends CoursesClient.HttpAuthentication {
```

Comment:
Yeah this is empty but that's ok at least for now, IMO.

On the other hand, I'd like to have `Authentication` objects in `model` as they really are part of our domain. And on the other hand, I wouldn't like `CoursesClient` to be dependent on `model` package, so this way it all works out quite okay-ishly.

Reply:
I had approximately the same thoughts, thanks for a cross-check-up! :bow: